README.md

```diff
@@ -13,18 +13,13 @@ The (internal) project-board is [here](https://github.com/orgs/hpi-sam/projects/
 3. [npm](https://www.npmjs.com/) should already come with NodeJs - if not install it
 4. Clone this repository
 5. Run `npm run setup` from the root folder
-6. (Optional) We have a list of recommended [vscode](https://code.visualstudio.com/) extensions. We strongly recommend you to use them if you are developing. You can see them via [the `@recommended` filter in the extensions panel](https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions).
+6. Choose whether you want to use a database, see [the next section](#setup---database)
```

Comment:
Instead of referencing this section here, put the points directly to this point.